src/knot_cloud.c

```diff
@@ -568,11 +516,14 @@ int knot_cloud_update_schema(const char *id, struct l_queue *schema_list)
 	 * Expiration
 	 *	2000 ms
 	 */
-	result = mq_publish_direct_message(MQ_EXCHANGE_DEVICE,
-					   MQ_CMD_SCHEMA_SENT,
-					   headers, 1,
-					   MQ_MSG_EXPIRATION_TIME_MS,
-					   json_str);
+	mq_message_data_t mq_message = {
+				 MQ_MESSAGE_TYPE_DIRECT,
+				 MQ_EXCHANGE_DEVICE, MQ_CMD_SCHEMA_SENT, 
```

Comment:
Could you remove the boundary white space at the end of the line?
```suggestion
				 MQ_EXCHANGE_DEVICE, MQ_CMD_SCHEMA_SENT,
```

Reply:
Yes, i'll do that